src/services/api/accountApi.js

```diff
@@ -48,10 +48,12 @@ app.factory('AccountApi', function ($q, Peers, Account) {
    * @param {Number} amount - A floating point value in LSK
    * @param {String} secret - account's primary passphrase
    * @param {String} [secondSecret = null] - The second passphrase of the account (if enabled).
+   * @param {String} [offsetSeconds = 10] - The offset time to compensate potential
+   *    lead/lag in client system's time
    */
   this.transactions.create = (recipientId, amount, secret,
-    secondSecret = null) => Peers.sendRequestPromise('transactions',
-      { recipientId, amount, secret, secondSecret });
+    secondSecret = null, offsetSeconds = 10) => Peers.sendRequestPromise('transactions',
+      { recipientId, amount, secret, secondSecret, offsetSeconds });
```

Comment:
Does this cover voting transactions, second sig reg etc? All tx need to use this new property.